west.yml

```diff
@@ -157,7 +157,7 @@ manifest:
       groups:
         - hal
     - name: hal_espressif
-      revision: 174547ef6a97dafcd6786ecd171cc701f5c0893b
```

Comment:
This change is pulling in a few more intermittent commits that were added prior to the assert fix. 
If we need this for 4.0, how can we limit the risk?